ElectronApp/src/__tests__/EditableTable.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+
+import App from '../renderer/App';
+
+describe('Editable Table', () => {
+  const leftClick = { button: 0 };
+
+  it('should be in document', () => {
+    render(<App />, { wrapper: MemoryRouter });
```

Comment:
I think we should scope this unit test to only apply to the EditableTable component. At the moment, this test provides coverage already provided by other tests - i.e. checking if the whole app renders and checking if the nav works. This can be an issue because this test could fail without anything being wrong with the EditableTable component.

It's good to have tests for how our modules work together, but first we need to test our modules in isolation. 

Since EditableTable is pretty simple at the moment, this test could just be:

`expect(render(<EditableTable />)).toBeTruthy();`